Makefile.devcontainer

```diff
@@ -75,6 +75,7 @@ FILES = $(shell find . -type f -name '*.go')
 
 format:                         ## Format source code.
 	goimports -local github.com/percona/pmm-managed -l -w $(FILES)
+	gofumpt -l -w $(FILES)
```

Comment:
Let's invoke first `gofumpt` and then `goimports`

Reply:
sure